packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js

```diff
@@ -74,6 +100,11 @@ export const TableRows = ({
     }
   };
 
+  // block rendering until the review stage component is fully loaded in EE
+  if (!ReviewWorkflowsStage) {
+    return null;
+  }
+
```

Comment:
This won't block rednering on content-types that don't have review workflows enabled 🤔?  Maybe also check `hasReviewWorkflows`?